homeassistant/components/litterrobot/sensor.py

```diff
@@ -225,3 +232,10 @@ def icon(self) -> str | None:
         if (icon := self.entity_description.icon_fn(self.state)) is not None:
             return icon
         return super().icon
+
+    @property
+    def last_reset(self) -> datetime | None:
+        """Return the time when the sensor was last reset, if any."""
+        if self.entity_description.key == "visits_today":
+            return dt_util.start_of_local_day()
+        return super().last_reset
```

Comment:
```suggestion
        if self.entity_description.key == "visits_today":
            return dt_util.start_of_local_day()
        return super().last_reset()
```
the `()` are missing, right?